frontend/src/components/resourceQuota/Details.tsx

```diff
@@ -0,0 +1,43 @@
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
+import ResourceQuota from '../../lib/k8s/resourceQuota';
+import { DetailsGrid, SimpleTable } from '../common';
+
+export default function ResourceQuotaDetails() {
+  const { namespace, name } = useParams<{ namespace: string; name: string }>();
+  const { t } = useTranslation();
```

Comment:
You should preload the namespaces you use in this view, I think this is why you may have warnings.
i.e. `useTranslation(['frequent', 'resourceQuota', ...]);`